packages/shared/src/entities/notification-template/notification-template.interface.ts

```diff
@@ -43,8 +43,8 @@ export interface IBlueprint extends INotificationTemplate {
 }
 
 export class StepIssues {
-  body?: Record<string, StepIssue>;
```

Comment:
Did not find a usecase for this. Might be saved in the DB though so might have to revert?

Reply:
You mean do a migration to clean it? IT will be eliminated when we eliminate message_templates